tests/unit/components/Page.spec.js

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai';
+import { mount } from '@vue/test-utils';
+import { Page, Label } from '../../../src/main';
```

Comment:
page and label should be imported from the original path. it should be removed from main.js

Reply:
Page and label imported from the original path